chroma_core/lib/util.py

```diff
@@ -50,6 +50,17 @@ def sizeof_fmt_detailed(num):
     return int(num)
 
 
+def target_label_split(label):
+    """
+    Split a target label into a tuple of it's parts: (fsname, target type, index)
```

Comment:
Should `target type` be `label`? I asked based on the line below:
```
(_, label, index) = target_label_split(t.get_label())
```

Reply:
I guess the `label` variable should be something like `target_type`